src/ui/toolbar/components/ToolbarColorPicker.vue

```diff
@@ -32,7 +32,7 @@
         class="c-swatch"
         :style="{ background: options.value }"
         role="img"
-        :aria-label="None"
+        aria-label="None"
```

Comment:
Driveby as it was throwing a warning every time the styles object was brought up.

Reply:
Nice catch